src/devices/Ads1115/PgaConfig.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ads1115
+{
+    /// <summary>
+    ///  Configure the Programmable Gain Amplifier
+    /// </summary>
+    public enum PgaConfig
+    {
+        /// <summary>
+        /// ±6.144V
+        /// </summary>
+        FS6144 = 0x00,
+        /// <summary>
+        /// ±4.096V
+        /// </summary>
+        FS4096 = 0x01,
+        /// <summary>
+        /// ±2.048V
+        /// </summary>
+        FS2048 = 0x02,
+        /// <summary>
+        /// ±1.024V
+        /// </summary>
+        FS1024 = 0x03,
+        /// <summary>
+        /// ±0.512V
+        /// </summary>
+        FS512 = 0x04,
```

Comment:
you might want to consider changing to FS0512 so that VS/intellisense sorts the values better